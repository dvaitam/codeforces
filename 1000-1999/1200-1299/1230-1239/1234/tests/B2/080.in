12 7
18 11 17 16 11 20 4 19 10 18 9 14
