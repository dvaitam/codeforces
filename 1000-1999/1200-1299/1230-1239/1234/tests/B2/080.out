7
14 9 18 10 19 4 20
